package.json

```diff
@@ -3,6 +3,7 @@
   "version": "10.2.1",
```

Comment:
Let's do a major version bump with these changes.

Reply:
I bumped it.

Reply:
Done.